glue/viewers/image/qt/slice_widget.py

```diff
@@ -92,7 +107,8 @@ def sync_sliders_from_state(self, *args):
                     world = world_axis(self.data.coords, self.data,
                                        pixel_axis=self.data.ndim - 1 - i,
                                        world_axis=self.data.ndim - 1 - i)
-                    world_unit = self.data.coords.world_axis_units[i]
+                    world_unit = self.data.coords.world_axis_units[self.data.ndim - 1 - i]
```

Comment:
This might be fixed in master (by @jsub1) so will need rebasing

Reply:
Rebased with the changes in master picked up

Reply:
Wait, think there was some problem with my rebase... let me fix it 😉

Reply:
Confirmed changes picked up...